signac_dashboard/modules/project_image_viewer.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import glob
+import itertools
+import os
+
+from flask import render_template
+
+from signac_dashboard.module import Module
+
+
+class ProjectImageViewer(Module):
```

Comment:
1. Charlotte and I decided a while back to keep the longer "JobContext" names to avoid user confusion over the number of things named job.
2. I refactored this so they use the same template too!
3. I still like the idea about supported contexts as they can apply to other modules.